cli/command/plugin/install.go

```diff
@@ -142,6 +142,7 @@ func acceptPrivileges(dockerCli command.Cli, name string) func(privileges types.
 		for _, privilege := range privileges {
 			fmt.Fprintf(dockerCli.Out(), " - %s: %v\n", privilege.Name, privilege.Value)
 		}
-		return command.PromptForConfirmation(dockerCli.In(), dockerCli.Out(), "Do you grant the above permissions?"), nil
+		ctx := context.TODO()
```

Comment:
here the context isn't apparent. Either we use the top level dockerCli context or create a new context within the function. The moby project does not support passing context on this function call https://github.com/moby/moby/blob/master/api/types/client.go#L292

Reply:
FWIW; if we would benefit from passing contexts, we should consider updating the moby code. For a lot of code in moby, `context` didn't exist yet when it was written, and we started passing through contexts more, but there's still many areas where we didn't.

Reply:
I'll take a look at also adding context in the moby api client. It will help greatly with otel and just task cleanup in general.